package.json

```diff
@@ -112,6 +112,7 @@
     "angular-eslint": "^19.4.0",
     "babel-loader": "^10.0.0",
     "eslint": "^9.24.0",
+    "eslint-config-prettier": "^10.1.5",
```

Comment:
I think this is not related to the issues in this PR. Please remove it here (and if it's an important change, open a separate PR to add eslint-config-prettier). What is this needed for?